sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -430,6 +430,7 @@ def check_verbosity(
     ],
 )
 @pytest.mark.parametrize("csr_container", CSR_CONTAINERS)
+@pytest.mark.thread_unsafe  # manually captured stdout
```

Comment:
Note: to avoid having to manually maintain those `thread_unsafe` annotation, we should use the `capsys` fixture instead of manually monkey patching `sys.stdout`.